docs/v3.x/deployment/ibm-cloud.md

```diff
@@ -0,0 +1,332 @@
+# IBM Cloud
+
+This is a step-by-step guide for deploying a Strapi project on [IBM Cloud](https://www.ibm.com/cloud). Among the available alternatives, this guide covers the deployment on [IBM Cloud Foundry](https://www.cloudfoundry.org/the-foundry/ibm-cloud-foundry) using the official databases services.
+
+### IBM Cloud Install Requirements
+
+- You must have a [free IBM Cloud account](https://cloud.ibm.com/registration) before doing these steps.
+
+If you already have the IBM Cloud CLI installed locally on your computer. Skip to [Login to IBM Cloud](#_2-login-to-ibm-cloud-from-your-cli).
+
+#### 1. IBM Cloud CLI Installation
+
+Download and install the [CLI](https://cloud.ibm.com/docs/cli) for your operating system:
+
+:::: tabs
+
+::: tab "Mac O/S"
+Run the following from your terminal:
+
+```bash
+curl -sL https://ibm.biz/idt-installer | bash
+```
+
+:::
+
+::: tab Ubuntu
+Run the following from your terminal:
+
+```bash
+curl -sL https://ibm.biz/idt-installer | bash
+```
+
+:::
+
+::: tab Windows
+Run the following in Powershell as administrator:
+
+```bash
+[Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls11, Tls, Ssl3"; iex(New-Object Net.WebClient).DownloadString('https://ibm.biz/idt-win-installer')
+```
+
+:::
+
+::::
+
+#### 2. Login to IBM Cloud from your CLI
+
+Next, you need to login to IBM Cloud from your computer.
+
+```bash
+ibmcloud login
+```
+
+Follow the instructions and return to your command line.
+
+#### 3. Create a new project (or use an existing one)
+
+Create a Strapi project with MongoDB [section of the documentation](../guides/databases.md#install-strapi-locally-with-mongodbb). But don't worry about the username and password for now. For convenience we´re going to use the URI instead.
```

Comment:
Should remove directly references to databases (as later on you state using --quickstart which is SQLite).

If IBM supports multiple database types (PostgreSQL, MySQL, MariaDB, or MongoDB) then eventually the docs are going to be updated to be more agnostic for all guides unless specific steps are required such as MongoDB Atlas

Reply:
I always use MongoDB, but I have also covered Postgre. I can check and document the rest supported by IBM Cloud once the documentation is simpler.

"Should remove directly references to databases (as later on you state using --quickstart which is SQLite)." -> I'm a bit lost here

Reply:
After re-reading through I see you are just using quickstart to get the structure of Strapi and later setup the database, can ignore this for now :P

Reply:
I also think it's a bit messy, but I think it's better to keep the consistency among the documents and we can make all simpler at the same time. :)